tidypolars/tidypolars.py

```diff
@@ -144,6 +144,39 @@ def clone(self):
         """Very cheap deep clone"""
         return super().clone().pipe(from_polars)
 
+    def count(self, *args, sort: bool = False, name: str = 'N'):
+        """
+        Returns row counts of the dataset. 
+        If bare column names are provided, count() returns counts by group.
+
+        Parameters
+        ----------
+        *args : Union[str, Expr]
+            Columns to find distinct/unique rows
+        sort : bool
+            Should columns be ordered in descending order by count
+        name : str
+            The name of the new column in the output. If omitted, it will default to N.
+
+        Examples
+        --------
+        >>> df = tp.Tibble({'a': range(3), 'b': ['a', 'a', 'b']})
+        >>> df.count()
+        >>> df.count('b')
+        """
+        #TODO: Add logic to arrange columns by distinct value by order in column
+        args = _args_as_list(args)
+        
```

Comment:
I don't know how to comment this on the correct line, but around line 69 there is a `__dir__()` method. Can you add `'count'` to the list in there?